openapi/paths/cashier-custom-proeprty-sets@{id}.yaml

```
@@ -0,0 +1,82 @@
+parameters:
+  - $ref: ../components/parameters/resourceId.yaml
+get:
+  x-products:
+    - Core
+  tags:
+    - Cashier
+  summary: Retrieve a cashier custom property set
+  operationId: GetCashierCustomPropertySet
+  x-sdk-operation-name: get
+  description: Retrieves a cashier custom property set with a specified ID.
```

suggestion: I think it reads better to move 'custom' before 'cashier'
```suggestion
  description: Retrieves a custom cashier property set with a specified ID.
```